referral-form-data-process/lib/sendDataToAPI.ts

```diff
@@ -0,0 +1,192 @@
+    clientThreeFirstName: undefined,
+    clientThreeLastName: undefined,
+    clientFourFirstName: undefined,
+    clientFourLastName: undefined,
+    clientFiveFirstName: undefined,
+    clientFiveLastName: undefined,
+    clientSixFirstName: undefined,
+    clientSixLastName: undefined,
+    clientSevenFirstName: undefined,
+    clientSevenLastName: undefined,
+    clientEightFirstName: undefined,
+    clientEightLastName: undefined,
+  };
+  let clientsValue: string[] | undefined = [];
+
+  const dataEntries = Object.entries(formData);
+  for (const [key, value] of dataEntries) {
+    if (key === "Referrer First Name") {
+      formDataAnswersObject.referrerFirstName = value.toString();
+    }
+    if (key === "Referrer Last Name") {
+      formDataAnswersObject.referrerLastName = value.toString();
+    }
+    if (
+      key === "What support do you think the child(ren) would benefit from?"
+    ) {
+      formDataAnswersObject.requestedSupport = value.toString();
+    }
+    if (key === "Child 1: Child's First Name") {
+      formDataAnswersObject.clientOneFirstName = value.toString();
+    }
+    if (key === "Child 1: Child's Last Name") {
+      formDataAnswersObject.clientOneLastName = value.toString();
+    }
+    if (key === "Child 2: Child's First Name") {
+      formDataAnswersObject.clientTwoFirstName = value.toString();
+    }
+    if (key === "Child 2: Child's Last Name") {
+      formDataAnswersObject.clientTwoLastName = value.toString();
+    }
+    if (key === "Child 3: Child's First Name") {
+      formDataAnswersObject.clientThreeFirstName = value.toString();
+    }
+    if (key === "Child 3: Child's Last Name") {
+      formDataAnswersObject.clientThreeLastName = value.toString();
+    }
+    if (key === "Child 4: Child's First Name") {
+      formDataAnswersObject.clientFourFirstName = value.toString();
+    }
+    if (key === "Child 4: Child's Last Name") {
+      formDataAnswersObject.clientFourLastName = value.toString();
+    }
+    if (key === "Child 5: Child's First Name") {
+      formDataAnswersObject.clientFiveFirstName = value.toString();
+    }
+    if (key === "Child 5: Child's Last Name") {
+      formDataAnswersObject.clientFiveLastName = value.toString();
+    }
+    if (key === "Child 6: Child's First Name") {
+      formDataAnswersObject.clientSixFirstName = value.toString();
+    }
+    if (key === "Child 6: Child's Last Name") {
+      formDataAnswersObject.clientSixLastName = value.toString();
+    }
+    if (key === "Child 7: Child's First Name") {
+      formDataAnswersObject.clientSevenFirstName = value.toString();
+    }
+    if (key === "Child 7: Child's Last Name") {
+      formDataAnswersObject.clientSevenLastName = value.toString();
+    }
+    if (key === "Child 8: Child's First Name") {
+      formDataAnswersObject.clientEightFirstName = value.toString();
+    }
+    if (key === "Child 8: Child's Last Name") {
+      formDataAnswersObject.clientEightLastName = value.toString();
+    }
```

Comment:
Instead of a huge if block that will only grow with the number of questions could we use an object to tidy/speed up this code.

I'm writing this in github comment so take it with a grain of salt but think something like

```
const mappingOfQuestionsToProperties = new Map(
    ["Child 1: Child's First Name", "clientOneFirstName"],
    [...],
)

if(mappingOfQuestionsToProperties.has(key) {
    formDataAnswersObject[mappingOfQuestionsToProperties.get(key)] = value.toString()
}

```